tests/bruno/e2e/udt/08-node1-add-tlc.bru

```diff
@@ -25,7 +25,7 @@ body:json {
         "channel_id": "{{N1N2_CHANNEL_ID}}",
         "amount": "{{payment_amount}}",
         "payment_hash": "{{payment_hash}}",
-        "expiry": 40
+        "expiry": "0x20000000000"
```

Comment:
The interpretation of this expiry number is a little bit muddy. Do we want the clients to pass time duration or absolute timestamp for the expiry time? Since we are using this number directly for the `AddTlc` message, current code deems this number as an absolute timestamp. This invalidates all of the tests here, as all of they are invalid parameters (0x20000000000 is already a time in the past).

No matter what meaning we decide to choose, current documentation should be changed accordingly. https://github.com/contrun/fiber/blob/4fdf0042a5af3878b069cbee84451f29b05e65a7/src/rpc/README.md?plain=1#L135